### ISSUE TITLE
Update mongoose version?

### ISSUE THREAD

**Issue by droidkid** [benas/gamehub.io#18]
On trying to run the project, I got the following error:
Error: Cannot find module 'mongodb/node_modules/bson'

This issue is also referenced here: https://github.com/Automattic/mongoose/issues/2718 (Although the fix there did not help me). Updating mongoose to 4.7.4 fixed the issue for me.

Has anybody else seen this error (or is it something to do with my system?) and know any other fixes? I could submit a pull request for updating the mongoose version if that's ok also.

Thanks!

**Comment by benas** [comment]
Hi thanks for reporting this! Strange behaviour.. Dependencies have not changed and npm install does not work anymore.

This project needs a fresh update for all components (mongo , node, elastic, etc..).
I need to find some time to update all dependencies to latest versions and make sure everything still work as expected.

In the meanwhile, you are welcome to open a PR with mongoose upgrade.
Many thanks upfront!

**Comment by smattiso** [comment]
Confirmed this still happens as of Apr 13, 2017.

**Comment by uouaks** [comment]
Hi, Ben! Do you plan to update this project?

**Comment by uouaks** [comment]
> Hi thanks for reporting this! Strange behaviour.. Dependencies have not changed and npm install does not work anymore.
> 
> This project needs a fresh update for all components (mongo , node, elastic, etc..).
> I need to find some time to update all dependencies to latest versions and make sure everything still work as expected.
> 
> In the meanwhile, you are welcome to open a PR with mongoose upgrade.
> Many thanks upfront!

Hi, Ben! Do you plan to update this project?

**Comment by benas** [comment]
Yes I do! This project seems to be dead but it is not! It is in my todolist to update it but this task has not the top priority for now. I'll add a note about this in the readme.
